egs/how2/st1/conf/tuning/train_pytorch_transformer.yaml

```diff
@@ -10,7 +10,7 @@ adim: 256
 aheads: 4
 
 # hybrid CTC/attention
-mtlalpha: 0.0  # this must be 0.0
+mtlalpha: 0.0
```

Comment:
Is it  OK? (looks different from other yaml)

Reply:
Transformer is not supported in ST modules (using ASR modules).

Reply:
Of course, I can modify this PR for Transformer, but I think it's too much in a single PR.

Reply:
Then, is it needed to add here?

Reply:
OK. Just keep for next PR.